### PR TITLE
feat: i18next initial setup

### PR DESCRIPTION
**Pull request by Skia69**
Added i18n.js in the src folder as well as a locales folder containing 3 files respectively: ar.json, en.json and fr.json

### CODE REVIEW

**Comment by Skia69** [comment]
I just wanna mention that further adjustments might be required to get this to work 100% 
But for the time being I believe this is the barebones setup

File: src/i18n.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import i18n from 'i18next';
```
Comment (adlogi) [inline src/i18n.js:1]: I think this file still needs to be imported in index.js

https://github.com/i18next/i18next-http-backend

**Review by adlogi — CHANGES_REQUESTED** [review]

**Comment by Skia69** [comment]
I imported the i18n.js file into index.js
I'm not sure if that's all

File: src/i18n.js (hunk 1 above)
Comment (Skia69) [reply src/i18n.js:1]: I imported the i18n.js file into index.js
I'm not sure if that's all

**Pull request merged**